### PR TITLE
one acepted pr, one fixed by developer

### PR DESCRIPTION
**Pull request by qz0610**
https://github.com/FasterXML/jackson-dataformats-binary/commit/7e3c86905f9d5f1fe8c8922810980a169dd953c1
This commit fixed the flakiness by changing avro version from 1.8.2 to 1.11.3. The flakiness can be reproduce by switching avro version back to 1.8.2.

### CODE REVIEW

**Comment by darko-marinov** [comment]
We need some logs as evidence.

**Comment by qz0610** [comment]
passes in 7e3c86905f9d5f1fe8c8922810980a169dd953c1 for `-DnondexRuns=100` :
[7e3c86905f9d5f1fe8c8922810980a169dd953c1.log](https://github.com/user-attachments/files/17428232/7e3c86905f9d5f1fe8c8922810980a169dd953c1.log)

fails in 7e3c86905f9d5f1fe8c8922810980a169dd953c1^:
[7e3c86905f9d5f1fe8c8922810980a169dd953c1^.log](https://github.com/user-attachments/files/17428233/7e3c86905f9d5f1fe8c8922810980a169dd953c1.log)

one accepted pr with three fixed flaky tests:
https://github.com/apache/tinkerpop/pull/2820

File: pr-data.csv [hunk 1]
```diff
@@ -3908,7 +3908,7 @@ https://github.com/FasterXML/jackson-dataformat-xml,2d08f8d0b0248a00c2c79b9a07e9
...
 https://github.com/FasterXML/jackson-dataformat-xml,2d08f8d0b0248a00c2c79b9a07e936b3287edb07,.,com.fasterxml.jackson.dataformat.xml.ser.TestSerializationAttr.testIssue6,OD,Accepted,https://github.com/FasterXML/jackson-dataformat-xml/pull/499,
 https://github.com/FasterXML/jackson-dataformat-xml,f5cc10a910b153ec9f162c6d212212b39bfc889d,.,com.fasterxml.jackson.dataformat.xml.ser.TestSerializationWithFilter.testPullRequest616,ID,,,
-https://github.com/FasterXML/jackson-dataformats-binary,d78d13c574a4b139438841bd5859b8b99beb63ea,avro,com.fasterxml.jackson.dataformat.avro.interop.annotations.AvroEncodeTest.testByteValue[Apache to Jackson with Apache schema],ID,,,https://github.com/FasterXML/jackson-dataformats-binary/issues/305
+https://github.com/FasterXML/jackson-dataformats-binary,d78d13c574a4b139438841bd5859b8b99beb63ea,avro,com.fasterxml.jackson.dataformat.avro.interop.annotations.AvroEncodeTest.testByteValue[Apache to Jackson with Apache schema],ID,DeveloperFixed,,https://github.com/FasterXML/jackson-dataformats-binary/commit/7e3c86905f9d5f1fe8c8922810980a169dd953c1
```
Comment (darko-marinov) [inline pr-data.csv:3911]: Please preserve the original issue with a semicolon (check other rows with a semicolon).

In the future, please don't put log files here on GitHub but `scp` to your VM and let us know the VM ID and the path (can use `~` to not show your NetID).

**Comment by darko-marinov** [comment]
Please squash the three commits (but don't close the PR). In the future, you may want to open separate PRs for separate cases (e.g., one `DeveloperFixed` vs. three `Accepted`).

**Pull request merged**